src/docs/asciidoc/modules/ROOT/pages/chapter3.adoc

```diff
@@ -94,10 +115,10 @@ This project has only a few files in it, as you can see by running the `tree` co
                     └── demo
                         └── DemoApplicationTests.java
 
-14 directories, 6 files
+16 directories, 10 files
 ----
 
-`DemoApplication.java` is the heart of this application; the file and class name are not relevant. What is relevant is the `@SpringBootApplication` annotation and the class's `public static void main` method.
+`DemoApplication.java` is the heart of this application; the file and class name are irrelevant. What is relevant is the `@SpringBootApplication` annotation and the class's `public static void main` method.
```

Comment:
May be worth adding this to the end of the sentence in case you reference this later in the book

> ... method colloquially known as the main method.

Reply:
Can you please add this as a suggestion?

Reply:
```suggestion
`DemoApplication.java` is the heart of this application; the file and class name are irrelevant. What is relevant is the `@SpringBootApplication` annotation and the class's `public static void main` method colloquially known as the main method.
```